Makefile

```diff
@@ -0,0 +1,2 @@
+deploy-base     :; forge script script/Deploy.s.sol:DeployBaseFull --rpc-url ${ETH_RPC_URL} --interactives 1 --broadcast --verify
+deploy-ethereum :; forge script script/Deploy.s.sol:DeployMainnetFull --rpc-url ${ETH_RPC_URL} --interactives 1 --broadcast --verify
```

Comment:
Should match other repo. https://github.com/marsfoundation/xchain-ssr-oracle/blob/master/Makefile#L1 . Instead of specifying the rpc url here you should do it inside the script: https://github.com/marsfoundation/xchain-ssr-oracle/blob/master/script/Deploy.s.sol#L73

It's better for multichain deploys.